octodns_dnsmadeeasy/__init__.py

```diff
@@ -135,18 +143,26 @@ def records(self, zone_name):
     def record_multi_delete(self, zone_name, record_ids):
         zone_id = self.domains.get(zone_name, False)
         path = f'/{zone_id}/records'
-        self._request('DELETE', path, params={'ids': record_ids})
+
+        # batch requests per our batch size
+        for i in range(0, len(record_ids), self.batch_size):
```

Comment:
This would be good to have as a private method and reuse it. Something like

```
def _batch_records(self, record_ids):
    for i in range(0, len(record_ids), self.batch_size):
        yield record_ids[i : i + self.batch_size]

for batch in self._batch_records(record_ids):
  self._request('DELETE', path, params={'ids': batch})
```

Reply:
I'll make this change.